workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
+
+    def get_form(self, *args, **kwargs):
+        """Disable fields that are pre-populated."""
+        form = super().get_form(*args, **kwargs)
+        for field in self.get_initial():
+            try:
+                form.fields[field].disabled = True
```

Comment:
>> This doesn't make autocomplete fields disabled.

> It works for me. Can you point out the form where it fails to disable a field?

When I go to `http://127.0.0.1:8000/workshops/memberships/add/?organization=435`, the field is disabled, but the arrow at the right side is enabled. I can click it and a list of organizations is displayed. Then, I can select one of them. Does it work in the same way on your machine?

>> What about leaving pre-populated fields enabled?

> I like having them disabled so that I do not accidently change them.

I'm :+1: for disabling them if only they're *really* disabled. :-)